src/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/Components/UploadPhotosForm.tsx

```diff
@@ -74,15 +74,15 @@ export const UploadPhotosForm: React.FC<UploadPhotosFormProps> = ({
     }
   }, [values.photos])
 
-  const onDrop = useCallback((acceptedFiles: File[]) => {
+  const onDrop = (acceptedFiles: File[]) => {
```

Comment:
I think using callback here leads to `values.photos` still pointing to an older object (after the last photo has been added). Because of this all new files are added but also all old files that have been removed are added again.

Reply:
good thinking of removing this. i think no need for optimising here

Reply:
Exactly. Even adjusting the `useCallback` could use to similar bugs in the future.